9_report/src/fig_error_v_flow.R

```diff
@@ -54,7 +59,7 @@ fig_error_v_flow <- function(fig_ind, config_fig_yml, preds_ind, agg_nwis_ind, r
                            axis.title = element_text(size = 15),
                            legend.position = 'none',
                            plot.margin = unit(c(1,3,1,1),'lines'))+
-                     annotation_custom(grob = textGrob(label = sites[1], hjust = 0, rot = 270),
+                     annotation_custom(grob = textGrob(label = site_labels[match(sites[1], names(site_labels))], hjust = 0, rot = 270),
```

Comment:
could this be simplified to `site_labels[sites[1]]` since the values in `sites` are the names by which `site_labels` is indexed?

Reply:
ah right, will change and push up